kubernetes_deploy/staging/ingress.yml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: laa-fala
   namespace: laa-fala-staging
 spec:
+  tls:
+  - hosts:
+    - laa-fala-staging.apps.cloud-platform-live-0.k8s.integration.dsd.io
```

Comment:
Do we need TLS for staging?

Reply:
We currently do have it on staging, this will maintain that. Also if it's free and can't see downside of having it

Reply:
By default, we redirect http to https (so as to enforce https access). You'll need it for all environments I'm afraid.

Reply:
Thank you, both! I wanted to know whether the changes for staging we were required and if yes, why, but I was not clear on this intent :sweat_smile: